core/src/block.rs

```diff
@@ -212,8 +212,7 @@ impl PendingBlock {
         self,
         height: u64,
         previous_block_hash: HashOf<VersionedCommittedBlock>,
-        view_change_proofs: ViewChangeProofs,
-        invalidated_blocks_hashes: Vec<HashOf<VersionedValidBlock>>,
```

Comment:
Because they aren't used. It might be so that I have to reintroduce it when adding handling for soft-forks, #1170. I didn't want to assume an a priori solution. You could for example allow the latest block to be swapped for another valid block then lean on block sync to bring lost peers back on to the main chain.